### PR TITLE
feat: rhtap: pre-step for health check

### PR DESCRIPTION
**Pull request by psturc**
https://issues.redhat.com/browse/RHTAP-1973

### CODE REVIEW

**Comment by psturc** [comment]
/pj-rehearse pull-ci-redhat-appstudio-build-service-main-build-service-e2e

**Comment by psturc** [comment]
/pj-rehearse pull-ci-redhat-appstudio-build-service-main-build-service-e2e

**Comment by psturc** [comment]
/pj-rehearse pull-ci-redhat-appstudio-build-service-main-build-service-e2e

File: ci-operator/step-registry/redhat-appstudio/health-check/redhat-appstudio-health-check-commands.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+set -o errexit
+set -o pipefail
+
+export GITHUB_TOKEN
+GITHUB_TOKEN=$(cat /usr/local/ci-secrets/redhat-appstudio-qe/github-token)
+
+cd "$(mktemp -d)"
+git clone --branch main https://github.com/redhat-appstudio/qe-tools .
+make build
+./qe-tools prowjob health-check --notify-on-pr=true --fail-if-unhealthy=false
```
Comment (psturc) [inline ci-operator/step-registry/redhat-appstudio/health-check/redhat-appstudio-health-check-commands.sh:13]: for now the pre-step will only inform the PR author about the outage, but **won't fail the job completely**

that behaviour is planned to be enabled after we evaluate this tool to be working reliably

**Review by jkopriva — APPROVED** [review]

**Comment by jkopriva** [comment]
/pj-rehearse pull-ci-redhat-appstudio-build-service-main-build-service-e2e

**Pull request merged**